WordPressKit/WordPressComRestApi.swift

```diff
@@ -154,6 +156,14 @@ open class WordPressComRestApi: NSObject {
         uploadSessionManager.session.invalidateAndCancel()
     }
 
+    public func authTokenEquals(token: String) -> Bool {
+        return token == oAuthToken
```

Comment:
I found this line a bit confusing because it's hard to tell which of `token` and `oAuthToken` is the one being considered.

Method signature like `authTokenEquals(token candidateToken: String)` might improve that? I dunno. What do you think?

Reply:
I agree with @nheagy's observation above.

I should note, however, that in my testing, I did not see this method invoked in any code paths. For a hot fix, I'd generally advocate a "less is more" approach, but the changes do not discernably impact the behavior.

Reply:
Ah yes, that's unused now. Originally I used a NSNotification to handle the invalid token, and this helped checked which token had been invalidated (without exposing the token). Now that it uses a callback, WPAccount can capture `self` when setting the callback and make sure the right account gets cleared. I'll delete this